### PR TITLE
Remove error logging

### PR DESCRIPTION
**Pull request by heinrisch**
API needs to update before this is merged: https://github.com/wrapp/api/pull/452

### CODE REVIEW

File: txhttptrace.py [hunk 1]
```diff
@@ -82,13 +79,7 @@ def wrapper(ctx, request, *args, **kwargs):
...
         def do_finally(param):
             ctx['took'] = time.time() - start
-            if 'error' in ctx:
-                msg = 'request failed'
-                if _log_errors_as_warnings:
-                    _logger.warning(msg, **ctx)
-                else:
-                    _logger.error(msg, **ctx)
-            elif _debug:
+            if _debug:
```
Comment (cortex) [inline txhttptrace.py]: won't this always log "request successful", even when it was not?

Comment (heinrisch) [inline txhttptrace.py]: That is true, I'll change that line

**Comment by voroninman** [comment]
I don't understand the purpose of this PR. Why we're removing the error logging?

**Comment by heinrisch** [comment]
@voroninman removing faulty logging. This library logs every exception as an error. It should be the service's responsibility to log actual errors.

This logging causes most errors to be printed twice, once by the service and once by this library.

**Comment by voroninman** [comment]
@heinrisch Makes perfect sense. I missed that I'm in `txhttptrace` repo. Looks good to me.

**Pull request merged**